### PR TITLE
namelist parameter for photolysis max solar zenith

### LINKED ISSUE #321: photolysis maximum solar zenith
Maximum solar zenith angle used in photochemistry should be the same for cam-chem and waccm.

### PR DESCRIPTION
**Pull request by fvitt**
This implements a namelist variable for maximum solar zenith angle for photolysis.  The maximum solar zenith for CAM-Chem is increased to the value used in WACCM.

This changes answers for CAM-Chem compsets only.

closes #321

### CODE REVIEW

**Comment by fvitt** [comment]
@dkinnison,  Are you able to review the changes in this pull request?

**Comment by dkinnison** [comment]
> @dkinnison, Are you able to review the changes in this pull request?

Yes - Dr. Vitt.

**Review by cacraigucar — APPROVED** [review]

**Review by gold2718 — APPROVED** [review]
Looks good!

File: bld/namelist_files/namelist_definition.xml [hunk 1]
```diff
@@ -6802,6 +6802,13 @@ Full pathname of cross section dataset for short wavelengh photolysis
 Default: set by build-namelist.
 </entry>
 
+<entry id="photo_max_zen" type="real" category="cam_chem"
+       group="chem_inparm" valid_values="" >
+Maximum zeniith angle (degrees) used for photolysis.
```
Comment (nusbaume) [inline bld/namelist_files/namelist_definition.xml:6807]: Typo here (`zenith`).

**Review by nusbaume — APPROVED** [review]
Looks good!  Just one minor typo.

**Review by peverwhee — APPROVED** [review]

**Pull request closed without merging**